cmd/wallet-web/src/pages/Credentials.vue

```diff
@@ -116,7 +115,9 @@ export default {
     let { user, token } = this.getCurrentUser().profile;
     this.username = this.getCurrentUser().username;
     let credentialManager = new CredentialManager({ agent: this.getAgentInstance(), user });
-    this.fetchAllCredentials(credentialManager.getAll(token));
+    this.fetchAllCredentials(
+      credentialManager.getAll(token, { collectionID: this.$route.params.vaultId })
```

Comment:
What happens if `this.$route.params.vaultId` is null? Is `credentialManager` going to return all credentials?

Reply:
Yes